src/reports/package/root/package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "accessibility-insights-report",
+    "version": "0.0.1",
+    "description": "Accessibility Insights Report",
+    "license": "MIT",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Microsoft/accessibility-insights-web"
+    },
+    "dependencies": {
```

Comment:
I curious about this list of dependency, some of them strike me as not needed for the report, e.g `applicationinsights-js` (are we sending telemetry from the report or when building the report?), `idb-keyval` (are we accessing indexed db -from the browser)... I guess this maybe indirect dependencies for the report, if that's so, it sounds like some refactoring list for the near feature to clean/split up the report dependencies so we can reduce this list of dependencies.

Reply:
We could trim these for sure.

Reply:
I pruned the list of dependencies to just the following:
```
        "axe-core": "3.3.2",
        "classnames": "^2.2.6",
        "lodash": "^4.17.15",
        "moment": "^2.24.0",
        "office-ui-fabric-react": "6.210.5",
        "react": "^16.12.0",
        "react-dom": "^16.12.0",
        "uuid": "^3.3.3"
```